assingment.playground/Contents.swift

```diff
@@ -29,13 +29,16 @@ public func occupyEmptySeat(index: Int, employee: String = "👩🏻‍💼") {
 }
 
 public func workIsLeftBy(employee: String) {
+    if workplace.seats.contains(employee) {
     for index in 0 ... workplace.seats.count - 1 {
```

Comment:
Please fix indentation, you can let Xcode take care of that, select given method and use shortcut `control` + `i`